tss-esapi/src/abstraction/transient.rs

```diff
@@ -733,7 +733,7 @@ impl TransientKeyContextBuilder {
             })?;
         if let (Some(old_session), _, _) = context.sessions() {
             context.set_sessions((Some(new_session), None, None));
-            context.flush_context(old_session.handle().into())?;
```

Comment:
How come the new session types can't have a `handle()` method?

Reply:
Of course they can I just thought it was nicer to be able to convert them into one. Even though I am not so found of the double conversion.

Reply:
Maybe it would be better to add a conversion directly to an ObjectHandle?

In the long term I hope to be able to get rid of all the ESYS handle types because they are not really correct. The specification says that ```flush_context``` should take a TPMI_DH_CONTEXT but I am not sure if the CONTEXT, OBJECT and ENTITY interface types makes much sense in ESAPI layer.